packages/typespec-test/test/authoring/generated/openapi/2022-05-15-preview/openapi.json

```diff
@@ -930,7 +930,7 @@
     },
     "Azure.Core.Foundations.InnerError": {
       "type": "object",
-      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/Microsoft/api-guidelines/blob/vNext/Guidelines.md#7102-error-condition-responses.",
+      "description": "An object containing more specific information about the error. As per Microsoft One API guidelines - https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors.",
```

Comment:
@jiaodi could you help investigate why this appears?

Reply:
refer to: https://github.com/Azure/typespec-azure/pull/2512/files